### PR TITLE
Fix some error with anvil in Windows

### PR DESCRIPTION
**Pull request by sandhilt**
Im find some bugs on Windows environment.

### CODE REVIEW

**Comment by sandhilt** [comment]
image

**Comment by sandhilt** [comment]
image

**Review by fabiooshiro — APPROVED** [review]
approving on faith!

File: internal/commons/handle_release.go [hunk 1]
```diff
@@ -67,11 +67,8 @@ func (a AnvilRelease) PlatformCompatible() (string, error) {
...
 
-	if goarch == "amd64" && goos == "windows" {
-		return a.FormatNameRelease("", goarch, goarch, ""), nil
-	}
-
-	if (goarch == "amd64" || goarch == "arm64") && (goos == "linux" || goos == "darwin") {
+	if (goarch == "amd64" && goos == "windows") ||
+		((goarch == "amd64" || goarch == "arm64") && (goos == "linux" || goos == "darwin")) {
 		return a.FormatNameRelease("", goos, goarch, ""), nil
 	}
```
Comment (fabiooshiro) [inline internal/commons/handle_release.go:74]: Could you please include the goos and goarch in the error message?

File: internal/commons/handle_release.go [hunk 2]
```diff
@@ -67,15 +67,12 @@ func (a AnvilRelease) PlatformCompatible() (string, error) {
...
 
-	return "", fmt.Errorf("anvil: platform not supported")
+	return "", fmt.Errorf("anvil: platform not supported: os = %s; arch = %s", goarch, goos)
```
Comment (fabiooshiro) [inline internal/commons/handle_release.go:75]: I added more info

**Pull request merged**